sklearn/tests/test_dummy.py

```diff
@@ -272,14 +296,14 @@ def test_median_strategy_multioutput_regressor():
     y_test = random_state.randn(20, 5)
 
     # Correctness oracle
-    est = DummyRegressor(strategy="median")
-    est.fit(X_learn, y_learn)
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.